services/galley/test/integration/API/Teams.hs

```diff
@@ -41,12 +41,15 @@ import UnliftIO (mapConcurrently, mapConcurrently_)
 
 tests :: IO TestSetup -> TestTree
 tests s =
-  testGroup
-    "Teams API"
+  testGroup "Teams API" $
     [ test s "create team" testCreateTeam,
       test s "create multiple binding teams fail" testCreateMulitpleBindingTeams,
       test s "create binding team with currency" testCreateBindingTeamWithCurrency,
       test s "create team with members" testCreateTeamWithMembers,
+      testGroup "List Team Members" $
+        [ test s "a member should be able to list their team" testListTeamMembersDefaultLimit,
+          test s "the list should be limited to the number requested" testListTeamMembersLimited
+        ],
```

Comment:
```suggestion
           test s "truncated list contains the has_more bit" testListTeamMembersTruncatedSaysSo
         ],
```

Reply:
Yeah, this is still pending, I have added it to the tasks: https://github.com/zinfra/backend-issues/issues/1186#issuecomment-591560134